docs/demos/usb-camera-demo-rpi4.md

```diff
@@ -110,7 +110,11 @@ You tell Akri what you want to find with an Akri Configuration, which is one of
 2. any additional device filtering
 3. an image for a Pod (that we call a "broker") that you want to be automatically deployed to utilize each discovered device
 
```

Comment:
@leolumicrosoft, looks like some of the changes from your other PR (https://github.com/project-akri/akri-docs/pull/43#pullrequestreview-1071368922) were accidentally pulled into this, namely the updates to the usb demos :)